frontend/root_words_ui/mockAPI.js

```diff
@@ -30,18 +43,28 @@ const seedData = {
     أَحَدٌ: "احد",
     الرَّحْمَـٰنِ: "رحم",
     الرَّحِيمِ: "رحم",
-  },
+    غَاسِقٍ: "غسق"},
 };
 
-function getChapterVerses(chapter_number) {
-  return seedData.chapters[chapter_number];
+export function getChapterName(chapter_number){
+  return seedData.chapters[chapter_number].name;
```

Comment:
I see what you're doing here, but think about how the API is going to respond to your requests. You most likely won't be making a request for each specific attribute, but rather you'll get a JSON object that will contain the attributes which you can then access the same way you are doing in this function (i.e. response.data.name)